src/components/transactions/Warnings/SupplyCapWarning.tsx

```diff
@@ -12,9 +12,12 @@ export const SupplyCapWarning = () => {
         <Trans>Supply amount is limited due to Supply Cap</Trans>
       </Typography>
       <Typography>
-        Amet minim mollit non deserunt ullamco est sit aliqua dolor do amet sint. Velit officia
-        consequat duis enim velit mollit.{' '}
-        <Link href="https://docs.aave.com/faq/">
+        <Trans>
+          Supply caps limit the amount of a certain asset that can be supplied to the Aave protocol.
+          This helps reducing exposure to the asset and mitigate attacks like infinite minting or
+          price oracle manipulation.
+        </Trans>
```

Comment:
You can encapsulate the link with Trans, no need to have guide in a separate Trans element

Reply:
Updated :+1: